core/themes/basis/css/skin.css

```diff
@@ -1226,3 +1226,24 @@ div.error:before {
   /* background-image: -webkit-linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
   /* background-image: linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
 }
+
+/* Promobox view on default home page */
```

Comment:
The term "Promobox" seems to be a leftover from an earlier state. Better change to "Card".